doc/manual/management_system.rst

```diff
@@ -70,11 +72,11 @@ If there are other experiments with higher priority (e.g. a high-priority timed
 Otherwise, ``pause()`` returns immediately.
 To check whether ``pause()`` would in fact *not* return immediately, use :meth:`artiq.master.scheduler.Scheduler.check_pause`.
 
-The experiment must place the hardware in a safe state and disconnect from the core device (typically, by using ``self.core.comm.close()``) before calling ``pause``.
+The experiment must place the hardware in a safe state and disconnect from the core device (typically, by calling ``self.coredevice.comm.close()`` from the kernel, which is equivalent to :meth:`artiq.coredevice.core.Core.close`) before calling ``pause()``.
```

Comment:
To fix my repeated confusion why we don't have `pause()` documented at all, [`Scheduler.pause()`](https://github.com/m-labs/artiq/blob/f8ceea20d07776f3c7f3c6b97d6f23bfc5ebd451/artiq/master/worker_impl.py#L90) should probably be a stub method in the "root" [`Scheduler`](https://github.com/m-labs/artiq/blob/f8ceea20d07776f3c7f3c6b97d6f23bfc5ebd451/artiq/master/scheduler.py#L382) that we can cross-ref and document.